kernel/sched/MLFQ.c

```
@@ -1011,7 +1011,7 @@ void ProcessExitStub() {
     __builtin_unreachable();
 }
 
-static __attribute__((visibility("hidden"))) uint32_t CreateSecureProcess(const char * name, void (*entry_point)(void), uint8_t privilege, uint32_t initial_flags) {
+uint32_t CreateSecureProcess(const char * name, void (*entry_point)(void), uint8_t privilege, uint32_t initial_flags) {
```

Clang (14.0.6)

[warning] 2 adjacent parameters of 'CreateSecureProcess' of similar type ('int') are easily swapped by mistake

(bugprone-easily-swappable-parameters)